app/src/main/res/values-ar/strings.xml

```diff
@@ -1077,5 +1077,6 @@
     <string name="visit_count">%1$s (%2$d %3$s)</string>
     <string name="two_strings">%s %s</string>
     <string name="three_strings">%s %s %s</string>
+    <string name="member_description">(%2$d زيارة) %1$s</string>
```

Comment:
![image](https://github.com/user-attachments/assets/0d282813-0798-4bde-a65e-eca459519552)

markdown has issues with the format but adjust the string so that it matches the screenshot

Reply:
This one it's not letting me do. I tried many different ways to try to make it match this format but it ends up pushing the %2 to the left side